hack/install.sh

```diff
@@ -46,13 +46,13 @@ has_pacman() {
 install_required_packages() {
   if $(has_apt_get); then
     $SUDO apt-get update -y
-    $SUDO apt-get install -y curl runc bridge-utils
+    $SUDO apt-get install -y curl runc bridge-utils iptables
   elif $(has_yum); then
     $SUDO yum check-update -y
-    $SUDO yum install -y curl runc
+    $SUDO yum install -y curl runc iptables
```

Comment:
Yum may not have a package named iptables. So I would leave this as it is, it doesn't seem related to Debian Bullseye.